ci/kind/test-upgrade-antrea.sh

```diff
@@ -135,7 +141,7 @@ $THIS_DIR/kind-setup.sh create kind --antrea-cni false --images "$IMAGES"
 # /tmp. See:
 #  * https://github.com/actions/virtual-environments/issues/1514
 #  * https://forum.snapcraft.io/t/interfaces-allow-access-tmp-directory/5129
-$ROOT_DIR/hack/generate-manifest.sh --kind | docker exec -i kind-control-plane dd of=/root/antrea-new.yml
+$ROOT_DIR/hack/generate-manifest.sh --kind --on-delete | docker exec -i kind-control-plane dd of=/root/antrea-new.yml
```

Comment:
I understand why we use `--on-delete` here, but maybe add a comment about it here in case someone else needs to update this file in the future.

Reply:
done